src/main/resources/assets/paths/tdrDraftDAR.yaml

```diff
@@ -0,0 +1,25 @@
+post:
+  summary: createDraftDataAccessRequest
+  description: Create Draft Data Access Request
+  parameters:
+    - name: identifiers
+      in: query
+      description: Dataset identifiers to be included in the Data Access Request (comma separated). e.g. DUOS-123456, DUOS-123457
+      required: true
+      schema:
+        type: string
+    - name: projectTitle
+      in: query
+      description: Project title for the new Data Access Request.
+      required: false
+      schema:
+        type: string
+  tags:
+    - TDR
+  responses:
+    201:
+      description: Returns the created draft Data Access Request
+    400:
+      description: Bad request, no valid dataset identifiers were provided
```

Comment:
I think we could return a 404 error if there were any dataset identifiers that we could not find.

Reply:
Updated response to 404.